Library/Homebrew/cmd/commands.rb

```diff
@@ -16,12 +16,12 @@ class CommandsCommand < Command
     options do
       command "commands"
       desc "Show a list of built-in and external commands."
-      option name: "quiet", desc: "list only the names of commands without the header." do
-        option name: "include-aliases", desc: "the aliases of internal commands will be included."
-        # option name: "foo", desc: "do foo"
+      option option: "quiet", desc: "list only the names of commands without the header." do
+        option option: "include-aliases", desc: "the aliases of internal commands will be included."
+        option option: "foo", desc: "do foo"
       end
-      # option name: "prune", value: "days", desc: "remove all cache files older than <days>."
-      # option name: "prune1", value: "days", desc: "remove all cache files older than <days>."
+      option option: "prune", value: "days", equal_sign_presence: true, desc: "remove all cache files older than <days>."
```

Comment:
> For the former case(i.e.--foo=bar), it (i.e. all the code inside respective commands) gets the value(i.e. bar) using ARGV.value, whilst for the later case(i.e. --foo bar), it gets the value(i.e. bar) using ARGV.next

We should be moving away from modifying and monkeypatching `ARGV` altogether. However, for the sake of parsing, the `bar`s in both `--foo=bar` and `--foo bar` should be considered *values* instead of trailing *arguments*.